contrib/monitored_resource_util/src/main/java/io/opencensus/contrib/monitoredresource/util/LabelKey.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.monitoredresource.util;
+
+/**
+ * {@link LabelKey} represents the associated property of a {@link MonitoredResource}.
+ *
+ * @since 0.13
+ */
+public enum LabelKey {
+  GcpClusterName("cluster_name"),
```

Comment:
Please use the string names similar to the enum name, e.g. gcp_cluster_name

Reply:
I created https://github.com/census-instrumentation/opencensus-specs/issues/105, we can discuss if we want to have prefixes for all properties. For now I'll keep these names consistent with specs.

Reply:
As I suggested earlier, that's why it is important to have these properties belong to their class.

public class GcpGkeContainer {
  String getZone();
}

That's how the specs were designed, with this model the property belongs to that resource.

Reply:
That makes sense to me. I just updated `MonitoredResource` to have the properties as methods.